apps/emqx_bridge/src/emqx_bridge_api.erl

```diff
@@ -390,40 +390,50 @@ schema("/nodes/:node/bridges/:id/operation/:operation") ->
 
 '/bridges'(post, #{body := #{<<"type">> := BridgeType, <<"name">> := BridgeName} = Conf0}) ->
     Conf = filter_out_request_body(Conf0),
-    case emqx_bridge:lookup(BridgeType, BridgeName) of
-        {ok, _} ->
-            {400, error_msg('ALREADY_EXISTS', <<"bridge already exists">>)};
-        {error, not_found} ->
-            case ensure_bridge_created(BridgeType, BridgeName, Conf) of
-                ok -> lookup_from_all_nodes(BridgeType, BridgeName, 201);
-                {error, Error} -> {400, Error}
-            end
+    case has_redact_value(Conf) of
+        false ->
+            case emqx_bridge:lookup(BridgeType, BridgeName) of
+                {ok, _} ->
+                    {400, error_msg('ALREADY_EXISTS', <<"bridge already exists">>)};
+                {error, not_found} ->
+                    case ensure_bridge_created(BridgeType, BridgeName, Conf) of
+                        ok -> lookup_from_all_nodes(BridgeType, BridgeName, 201);
+                        {error, Error} -> {400, Error}
+                    end
+            end;
+        _ ->
+            {400, error_msg('BAD_REQUEST', <<"There is obfuscated sensitive data">>)}
```

Comment:
if this POST is for creation only, then it's reasonable to return 400 if `******` is provided as password.
otherwise if this also used for updating existing resource, the handler should replace it with the currently configured password.

Reply:
There is and only POST will create a bridge, so there is no need to handle with config updates here

Reply:
then there should be no need to check it here, because `<<"******">>` can be a legit password.